README.md

```diff
@@ -14,15 +14,35 @@ With `kubicorn` a user can create new clusters, modify and scale them, and take
 **NOTE:** This is a work-in-progress, we do not consider it production ready.
 Use at your own risk and if you're as excited about it as we are, maybe you want to join us on the #kubicorn channel in the Gophers Slack community.
 
+## Installing
+
+### Quickstart
+
+Assuming you have a local Go environment The Simplest method to install is to just run:
+
 ```
 $ go get github.com/kris-nova/kubicorn
 ```
 
+### Not so Quickstart
```

Comment:
Just saw your comment about pushing this to another doc 😆 

I vote doing that in another PR and letting this one in.